azure-devops/azext_devops/dev/team/wiki.py

```diff
@@ -137,16 +139,15 @@ def add_page(wiki, path, comment=_DEFAULT_PAGE_ADD_MESSAGE, content=None, file_p
     parameters = WikiPageCreateOrUpdateParameters()
     if content:
         parameters.content = content
-    if file_path:
-        fp = open(file_path, mode='r')
-        parameters.content = fp.read()
-        fp.close()
+    elif file_path:
+        from azext_devops.dev.common.utils import read_file_content
+        parameters.content = read_file_content(file_path=file_path, encoding=encoding)
     return wiki_client.create_or_update_page(parameters=parameters, wiki_identifier=wiki,
                                              project=project, path=path, version=None, comment=comment)
 
 
 def update_page(wiki, path, version, comment=_DEFAULT_PAGE_UPDATE_MESSAGE, content=None, file_path=None,
-                organization=None, project=None, detect=None):
+                encoding='utf-8', organization=None, project=None, detect=None):
```

Comment:
this will need some change in the example as well?

Reply:
Not really.. since this has a default value. Current examples will continue to work with copy paste.